src/components/checkbox/checkbox.html

```diff
@@ -1,33 +1,43 @@
 <div class="m-checkbox"
-      :class="{ 'm--is-checkbox-left': hasCheckboxLeft,
+     :class="{ 'm--is-checkbox-left': hasCheckboxLeft,
                 'm--is-checkbox-right': !hasCheckboxLeft,
                 'm--is-checked': propValue,
                 'm--is-disabled': isDisabled,
+                'm--is-readonly': readonly,
                 'm--is-focus': isFocus,
                 'm--has-error': hasError,
                 'm--is-valid': isValid }">
-    <input ref="checkbox"
+    <input v-if="!readonly"
+           ref="checkbox"
            type="checkbox"
            :id="id"
            class="m-checkbox__hidden"
            :disabled="isDisabled"
            v-model="propValue"
            @click="onClick"
            @focus="setFocus(true)"
-           @blur="setFocus(false)">
-    <label class="m-checkbox__wrap" :for="id" :aria-hidden="!hasLabelSlot">
-        <span class="m-checkbox__input" aria-hidden="true">
+           @blur="setFocus(false)" />
+    <input v-if="readonly"
+           type="hidden"
```

Comment:
Au lieu de dupliquer l'input, est-ce qu'il y aurait moyen d'en faire un qui est dynamique?

Reply:
J'y ai pensé. Au début j'avais mis un computed sur le type qui retournait hidden ou checkbox si on est readonly ou non. Mais il y a des attributs a enlever et d'autres à ajouter lorsque l'on passe d'un type à l'autre. Entre au remplacer v-model par value. Est ce que tu connais une façon de faire ça?

Reply:
Pour info, je viens de faire le test avec un seul input. J'ai une erreur à la compile, un conflit entre v-model et :value
template syntax error :value="propValue" conflicts with v-model on the same element because the latter already expands to a value binding internally

Reply:
@setur52 tu peux résoudre la conversation si il n'y a pas d'autres options, merci 1!